homeassistant/components/stookwijzer/strings.json

```diff
@@ -7,16 +7,25 @@
           "location": "[%key:common::config_flow::data::location%]"
         }
       }
+    },
+    "error": {
+      "unknown": "[%key:common::config_flow::error::unknown%]"
     }
   },
   "entity": {
     "sensor": {
+      "air_quality_index": {
+        "name": "Air Quality Index"
+      },
       "stookwijzer": {
         "state": {
-          "blauw": "Blue",
-          "oranje": "Orange",
-          "rood": "Red"
+          "code_yellow": "Code yellow",
```

Comment:
outside the scope of this PR, but there is a typo in "Select the location you want to recieve the Stookwijzer information for."

should probably be:
"Select the location for which you want to receive the Stookwijzer information."

Reply:
I like your suggestion. Thanks.